package.json

```
@@ -1,12 +1,14 @@
 {
     "name": "@brightlayer-ui/colors",
-    "version": "4.0.0-alpha.1",
+    "version": "4.0.0-alpha.2",
     "description": "Brightlayer UI colors for Eaton applications",
-    "main": "./dist/commonjs/index.js",
-    "types": "./dist/commonjs/index.d.ts",
+    "main": "./dist/index.js",
+    "types": "./dist/index.d.ts",
     "scripts": {
         "test": "./buildTest.sh",
-        "build": "yarn && tsc --module commonjs --outDir dist/commonjs && tsc --module ES2015 --outDir dist/es2015",
+        "build": "yarn tsc",
+        "build:colors-branding": "yarn && bash ./buildBrandingColors.sh",
+        "build:all": "yarn build && yarn build:colors-branding",
```

I don't think the `mkdir` step is required since you're moving to a folder that already exists (`./`).
```suggestion
        "clear:build": "rm -rf dist && rm -rf branding",
        "move:branding": "cp -r ./dist/branding branding && rm -rf ./dist/branding",
        "build": "yarn clear:build && yarn tsc && yarn move:branding",
```